rhizome/postgres/bin/install-wal-g

```diff
@@ -0,0 +1,24 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../../common/lib/util"
+
+# Install dependencies
+r "add-apt-repository -y ppa:longsleep/golang-backports"
+r "apt-get update"
+r "apt-get -y install golang-go cmake"
+
+r "mkdir -p var/wal-g"
+Dir.chdir("var/wal-g") do
+  # Fetch wal-g
+  r "git init"
+  r "git remote remove origin || true"
+  r "git remote add origin https://github.com/wal-g/wal-g.git"
+  r "git fetch origin --depth 1 c56a2315d3a63560f0227cb0bf902da8445963c7"
```

Comment:
is it possible that this will change over time? If so, how frequently? It may be a good idea to move this script into controlplane to avoid dataplane migrations.

Reply:
or maybe the commit id can be an argument?